kotsadm/web/src/components/SecureAdminConsole.jsx

```diff
@@ -192,10 +196,12 @@ class SecureAdminConsole extends React.Component {
       // strip quotes from token (golang adds them when the cookie value has spaces, commas, etc..)
       const loginData = {
         token: token.replace(/"/g, ""),
+        sessionRoles: Utilities.getCookie("session_roles"),
       };
       const loggedIn = await this.completeLogin(loginData);
       if (loggedIn) {
         Utilities.removeCookie("token");
+        Utilities.removeCookie("roles");
```

Comment:
Where is this cookie being set? should this be removing the `session_roles` cookie?

Reply:
oh, good catch, yes